src/toil/utils/toilMain.py

```diff
@@ -32,12 +33,10 @@ def main():
                 del sys.argv[1]
                 module.main()
 
-
 def loadModules():
     # noinspection PyUnresolvedReferences
-    from toil.utils import toilKill, toilStats, toilStatus, toilClean, toilLaunchCluster, toilDestroyCluster, toilSSHCluster, toilRsyncCluster
-    commandMapping = {name[4:].lower(): module for name, module in iteritems(locals())}
-    commandMapping = {name[:-7]+'-'+name[-7:] if name.endswith('cluster') else name: module for name, module in iteritems(commandMapping)}
+    from toil.utils import toilKill, toilStats, toilStatus, toilClean, toilLaunchCluster, toilDestroyCluster, toilSshCluster, toilRsyncCluster, toilDebugFile, toilDebugJob
+    commandMapping = { "-".join(map(lambda x : x.lower(), re.findall('[A-Z][^A-Z]*', name))) : module for name, module in iteritems(locals())}
```

Comment:
Nitpick: this is cool and compact, but not very readable.

Reply:
Tried to make it more readable.